lib/maths/analytics/unittest/CBoostedTreeTest.cc

```diff
@@ -1618,12 +1620,8 @@ BOOST_AUTO_TEST_CASE(testMseIncrementalAddNewTrees) {
               << ", 5 new trees = " << testError5 << ", 10 new trees = " << testError10);
     // The initial model has too little capacity so we should get substantial
     // improvements for adding trees.
-    BOOST_TEST_REQUIRE(0.95 * testError0 >= testError5);
+    BOOST_TEST_REQUIRE(0.9 * testError0 >= testError5);
     BOOST_TEST_REQUIRE(0.9 * testError0 >= testError10);
```

Comment:
I guess, in this form we could remove `testError10` altogether to simplify.

Reply:
Actually good point. I think 5 vs 10 is just a fragile. This has failed for more under perturbations before. You have to arrange for the necessary capacity to be just right. Perhaps if we overrode more hyperparameter values we could make it more reliable, but I think 0 vs 5 is sufficient if you're happy?